ccdb5_ui/templates/ccdb5_ui/ccdb_base.html

```diff
@@ -0,0 +1,227 @@
+    <!--[if lt IE 9]>
+    <link rel="stylesheet" href="/static/nemo/_/c/eot.css">
+    <![endif]-->
+    <link rel="stylesheet" href="/static/nemo/_/c/styles.css{{ STATIC_VERSION }}">
+
+    {% endblock %}
+
+    {% block app_css %}
+    {% endblock %}
+
+    {% block page_css %}
+    {% endblock %}
+
+    <!--[if lt IE 9]>
+    <link rel="stylesheet" href="{% static 'css/header.nonresponsive.css' %}"/>
+    <link rel="stylesheet" href="{% static 'css/footer.nonresponsive.css' %}"/>
+    <![endif]-->
+
+    <!--[if gt IE 8]><!-->
+    <link rel="stylesheet" href="{% static 'css/header.css' %}"/>
+    <link rel="stylesheet" href="{% static 'css/footer.css' %}"/>
+    <!--<![endif]-->
+
+<!--
+    ============
+    HEAD SCRIPTS
+    ============
+    The following scripts must be included in the head. DO NOT ADD ANY MORE.
+    All other scripts should go before the closing body tag.
+    If you come across a script that makes a convincing case to be included in
+    the head, then file an issue or PR to discuss including it.
+-->
+
+    <script src="{% static 'js/modernizr.min.js' %}"></script>
+    <!--[if lt IE 9]>
+    <script src="{% static 'nemo/_/js/html5shim.js' %}"></script>
+    <![endif]-->
+
+    <!--[if lt IE 9]>
+    <script>
+        // If in IE8 reverse no-js/js class change made by modernizr.
+        var docElement = document.documentElement;
+        docElement.className = docElement.className.replace( /(^|\s)js(\s|$)/, '$1no-js$2' );
+    </script>
+    <![endif]-->
+
+    <!--[if IE 9]><script src="{% static 'js/ie/common.ie.js' %}"></script><![endif]-->
+
+    <script src="/static/nemo/_/js/jquery-1.9.1.min.js{{ STATIC_VERSION }}"></script>
+
+    {% block analytics_js %}
+    {% endblock %}
+</head>
+
+<body class="{% block body_class %}page{% endblock %}" data-lang="{% trans "en" %}">
+{% block gtm_js %}
+<!-- Google Tag Manager -->
+{# Should always be included immediately after the opening <body> tag #}
+<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-KMMLRS"
+height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
+<script>(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':
+new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],
+j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src=
+'//www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);
+})(window,document,'script','dataLayer','GTM-KMMLRS');</script>
+<!-- End Google Tag Manager -->
+{% endblock %}
+
+    {% global_include "on-demand/header.html" %}
+    <div id="page">
+        {% block app_notification %}
+        {% endblock %}
+        <div id="wrapper">
+        <section id="maincontent">
+            <ul class="bread meta">
+                {% block breadcrumbs %}
+                {% endblock %}
+            </ul> <!-- /class=bread -->
+                {% block content %}
+                {% endblock %}
+                <!-- {% block share_box %}
```

Comment:
This is the deprecated share block - required cloning the legacy cfgov `base_update.html` template from legacy and removing.